packages/perspective-jupyterlab/src/config/plugin.config.js

```diff
@@ -15,7 +15,7 @@ module.exports = {
     entry: "./src/ts/index.ts",
     resolveLoader: {
         alias: {
-            "file-worker-loader": "@jpmorganchase/perspective/src/loader/file_worker_loader.js"
+            "file-worker-loader": "@jpmorganchase/perspective-webpack-plugin/lib/file-worker-loader.js"
```

Comment:
Please keep these packages consistent `lib` vs `src` - there's no reason to re-culture every package we add.